assets/header-templates/MPL-2.0.txt

```diff
@@ -0,0 +1,3 @@
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at https://mozilla.org/MPL/2.0/.
```

Comment:
I can confirm the content of this header matches the official version, https://www.mozilla.org/en-US/MPL/headers/